docs/snippets/publicity.rst

```diff
@@ -10,7 +10,9 @@ Publicity for all constructs is determined as follows: a construct is
 considered *public* if -
 
 1. Its immediate parent is public *and*
```

Comment:
Should we also update the docs to include our new behavior?

Reply:
Which new behavior are you referring to?

Reply:
Ah - I meant the behavior around syspath. That may catch some people off guard.

Reply:
It's kind of an internal implementation detail, but I can make mention of it as a "Note" or something near the bottom of the "How publicity is determined" section.

> **Note**, a construct's parent is recursively checked upward until we reach a directory in `sys.path` to avoid considering the complete filepath.

Thoughts?

Reply:
Let's keep this comment to what you suggested for now.

Reply:
Updated in commit https://github.com/PyCQA/pydocstyle/pull/494/commits/b72179dff7feec5e0880414d7e56a42b15b3a952.